continuous_integration/recipes/distributed/meta.yaml

```diff
@@ -74,7 +74,7 @@ outputs:
         - zict >=0.1.3
         - setuptools <60.0.0
       run_constrained:
-        - distributed-impl * {{ build_ext }}
+        - distributed-impl * {{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}_{{ build_ext }}
```

Comment:
Do we want to pin it this tightly?

Reply:
We don't need to, but I figured we may as well since we are generating unique `distributed-impl` packages for every `distributed` nightly - I can loosen this if you think that would be safer

Reply:
Maybe we can do the same thing here as we do with `dask-core`?

Reply:
Done in https://github.com/dask/distributed/pull/5867/commits/754daf199469884d38daaa0d95ea1a7bad731862